src/main/java/io/logz/logback/LogzioLogbackAppender.java

```diff
@@ -213,6 +219,16 @@ public void setGcPersistedQueueFilesIntervalSeconds(int gcPersistedQueueFilesInt
         this.gcPersistedQueueFilesIntervalSeconds = gcPersistedQueueFilesIntervalSeconds;
     }
 
+    public void setExceedMaxSizeAction(String exceedMaxSizeAction) {
+        if (!Arrays.asList("cut", "drop").contains(exceedMaxSizeAction.toLowerCase())) {
+            addWarn("The value for parameter ExceedMaxSizeAction is invalid, using default: cut");
+            this.exceedMaxSizeAction = "cut";
+        } else {
+
```

Comment:
can you add a log that say something like "invalid ExceedMaxSizeAction - falling back to option - cut " or something similar

Reply:
I did, line 224.